onetl/strategy/hwm_strategy.py

```diff
@@ -57,28 +54,70 @@ def enter_hook(self) -> None:
 
     def fetch_hwm(self) -> None:
         class_name = self.__class__.__name__
-
-        if self.hwm:
-            hwm_store = HWMStoreStackManager.get_current()
-
-            log.info("|%s| Loading HWM from %s:", class_name, hwm_store.__class__.__name__)
-            log_with_indent(log, "qualified_name = %r", self.hwm.qualified_name)
-
-            result = hwm_store.get_hwm(self.hwm.qualified_name)
-
-            if result is not None:
-                self.hwm = result  # type: ignore
-                log.info("|%s| Got HWM:", class_name)
-                self._log_hwm(self.hwm)
-            else:
-                log.warning(
-                    "|%s| HWM does not exist in %r. ALL ROWS/FILES WILL BE READ!",
-                    class_name,
-                    hwm_store.__class__.__name__,
-                )
-        else:
+        if not self.hwm:
             # entering strategy context, HWM will be set later by DBReader.run or FileDownloader.run
-            log.debug("|%s| HWM will not be loaded, skipping", class_name)
+            log.debug("|%s| HWM will not be fetched, skipping", class_name)
+            return
+
+        hwm_store = HWMStoreStackManager.get_current()
+        log.info("|%s| Fetching HWM from %s:", class_name, hwm_store.__class__.__name__)
+        log_with_indent(log, "name = %r", self.hwm.qualified_name)
+
+        result = hwm_store.get_hwm(self.hwm.qualified_name)
```

Comment:
```suggestion
        log_with_indent(log, "name = %r", self.hwm.name)

        result = hwm_store.get_hwm(self.hwm.name)
```
у нас в новых hwm же нет уже `qualified_name`?

Reply:
Я в отдельном PR буду обновлять etl-entities до 2.1, там и атрибуты переименую. А то и так уже много изменений